tests/TestHarness/Node.py

```diff
@@ -82,26 +82,13 @@ def __init__(self, host, port, nodeId: int, data_dir: Path, config_dir: Path, cm
         self.configureVersion()
 
     def configureVersion(self):
-        if 'v2' in self.nodeosVers:
-            self.fetchTransactionCommand = lambda: "get transaction"
-            self.fetchTransactionFromTrace = lambda trx: trx['trx']['id']
-            self.fetchBlock = lambda blockNum: self.processUrllibRequest("chain", "get_block", {"block_num_or_id":blockNum}, silentErrors=False, exitOnError=True)
-            self.fetchKeyCommand = lambda: "[trx][trx][ref_block_num]"
-            self.fetchRefBlock = lambda trans: trans["trx"]["trx"]["ref_block_num"]
-            self.fetchHeadBlock = lambda node, headBlock: node.processUrllibRequest("chain", "get_block", {"block_num_or_id":headBlock}, silentErrors=False, exitOnError=True)
-            self.cleosLimit = ""
-
-        else:
-            self.fetchTransactionCommand = lambda: "get transaction_trace"
-            self.fetchTransactionFromTrace = lambda trx: trx['id']
-            self.fetchBlock = lambda blockNum: self.processUrllibRequest("trace_api", "get_block", {"block_num":blockNum}, silentErrors=False, exitOnError=True)
-            self.fetchKeyCommand = lambda: "[transaction][transaction_header][ref_block_num]"
-            self.fetchRefBlock = lambda trans: trans["block_num"]
-            self.fetchHeadBlock = lambda node, headBlock: node.processUrllibRequest("chain", "get_block_info", {"block_num":headBlock}, silentErrors=False, exitOnError=True)
-            if 'v3.1' in self.nodeosVers:
-                self.cleosLimit = ""
-            else:
-                self.cleosLimit = "--time-limit 999"
+        self.fetchTransactionCommand = lambda: "get transaction_trace"
+        self.fetchTransactionFromTrace = lambda trx: trx['id']
+        self.fetchBlock = lambda blockNum: self.processUrllibRequest("trace_api", "get_block", {"block_num":blockNum}, silentErrors=False, exitOnError=True)
+        self.fetchKeyCommand = lambda: "[transaction][transaction_header][ref_block_num]"
+        self.fetchRefBlock = lambda trans: trans["block_num"]
+        self.fetchHeadBlock = lambda node, headBlock: node.processUrllibRequest("chain", "get_block_info", {"block_num":headBlock}, silentErrors=False, exitOnError=True)
```

Comment:
Seems like we could remove the lambda altogether. Not need for this indirection.

Reply:
I meant we could remove all of these lambdas. You will need to update the usage of the lambdas with the actual code.

Reply:
https://github.com/AntelopeIO/spring/pull/952/commits/bd5c628b9b6a06f857a084b08efcf0debdf76149